### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by alban**
Links were fixed With help from:
https://github.com/remarkjs/remark-validate-links

---

Before the patch:

<details>

```
$ remark -u validate-links .
CHANGELOG.md: no issues found
CONTRIBUTING.md: no issues found
README.md: no issues found
issue-management.md: no issues found
milestones.md
      22:3-22:124  warning  Link to unknown file: `semantic-conventions.md`                                                                missing-file             remark-validate-links

specification/api-distributedcontext.md: no issues found
specification/api-metrics.md
         8:3-8:19  warning  Link to unknown heading: `meater`. Did you mean `meter`                                                        missing-heading          remark-validate-links
       23:5-23:36  warning  Link to unknown heading: `type--counter`. Did you mean `type-counter`                                          missing-heading          remark-validate-links
       24:5-24:32  warning  Link to unknown heading: `type--gauge`. Did you mean `type-gauge`                                              missing-heading          remark-validate-links
      65:50-66:42  warning  Link to unknown file: `../semantic-conventions.md`. Did you mean `data-semantic-conventions.md`                missing-file             remark-validate-links

specification/api-propagators.md: no issues found
specification/api-resources.md: no issues found
specification/api-sampling.md: no issues found
specification/api-tracing.md
       24:7-24:30  warning  Link to unknown heading: `startspan`                                                                           missing-heading          remark-validate-links
       51:7-51:40  warning  Link to unknown heading: `getattributes-1`. Did you mean `getattributes`                                       missing-heading          remark-validate-links
    238:25-238:54  warning  Link to unknown heading: `setattribute`. Did you mean `set-attributes`                                         missing-heading          remark-validate-links
    239:13-239:32  warning  Link to unknown heading: `addlink`. Did you mean `add-links`                                                   missing-heading          remark-validate-links
    240:25-240:46  warning  Link to unknown heading: `addevent`. Did you mean `add-events`                                                 missing-heading          remark-validate-links
    333:55-334:38  warning  Link to unknown file: `semantic-conventions.md`. Did you mean `data-semantic-conventions.md`                   missing-file             remark-validate-links
    358:55-359:32  warning  Link to unknown file: `semantic-conventions.md`. Did you mean `data-semantic-conventions.md`                   missing-file             remark-validate-links

specification/concurrency.md: no issues found
specification/data-semantic-conventions.md: no issues found
specification/library-guidelines.md
       7:85-7:111  warning  Link to unknown file: `README.md`                                                                              missing-file             remark-validate-links

specification/library-layout.md: no issues found
specification/overview.md
    181:28-181:75  warning  Link to unknown file: `semantic-conventions.md`. Did you mean `data-semantic-conventions.md`                   missing-file             remark-validate-links

specification/performance.md: no issues found
specification/sdk-exporter.md: no issues found
work_in_progress/gRPC/gRPC.md: no issues found
work_in_progress/opencensus/HTTP-stats.md: no issues found
work_in_progress/opencensus/HTTP.md
  116:108-116:164  warning  Link to unknown file: `../../semantic-conventions.md`                                                          missing-file             remark-validate-links

work_in_progress/opencensus/StandardResources.md
       15:6-15:25  warning  Link to unknown heading: `cluster`                                                                             missing-heading          remark-validate-links
     31:61-31:116  warning  Link to unknown heading in `../specification/resource/Resource.md`: `merging`                                  missing-heading-in-file  remark-validate-links
     46:62-46:117  warning  Link to unknown heading in `../specification/resource/Resource.md`: `merging`                                  missing-heading-in-file  remark-validate-links
       62:1-62:56  warning  Link to unknown heading in `../specification/resource/Resource.md`: `merging`                                  missing-heading-in-file  remark-validate-links

work_in_progress/opencensus/gRPC-stats.md: no issues found
work_in_progress/opentracing/semantic_conventions.md
    17:108-17:164  warning  Link to unknown file: `../../semantic-conventions.md`. Did you mean `semantic_conventions.md`                  missing-file             remark-validate-links

work_in_progress/specification/README.md
      15:14-15:52  warning  Link to unknown file: `../../terminology.md`                                                                   missing-file             remark-validate-links
       40:1-40:39  warning  Link to unknown file: `../../terminology.md`                                                                   missing-file             remark-validate-links
    142:14-142:57  warning  Link to unknown file: `../../specification/tracing-api.md`. Did you mean `../../specification/api-tracing.md`  missing-file             remark-validate-links

work_in_progress/specification/encodings/BinaryEncoding.md: no issues found
work_in_progress/specification/encodings/CensusServerStatsEncoding.md: no issues found
work_in_progress/specification/encodings/README.md: no issues found
work_in_progress/specification/resource/Resource.md: no issues found
work_in_progress/specification/trace/LogCorrelation.md
      26:40-26:74  warning  Link to unknown heading in `Span.md`: `traceid`                                                                missing-heading-in-file  remark-validate-links
      30:39-30:71  warning  Link to unknown heading in `Span.md`: `spanid`                                                                 missing-heading-in-file  remark-validate-links
       35:1-35:49  warning  Link to unknown heading in `Span.md`: `supported-bits`                                                         missing-heading-in-file  remark-validate-links

work_in_progress/specification/trace/README.md
         8:3-8:30  warning  Link to unknown file: `gRPC.md`                                                                                missing-file             remark-validate-links
         9:3-9:30  warning  Link to unknown file: `HTTP.md`                                                                                missing-file             remark-validate-links

work_in_progress/specification/trace/Sampling.md: no issues found
work_in_progress/specification/trace/Span.md
        4:37-4:78  warning  Link to unknown file: `../../../terminology.md`                                                                missing-file             remark-validate-links

work_in_progress/specification/trace/TraceConfig.md: no issues found
work_in_progress/typedspans/README.md
      45:41-45:83  warning  Link to unknown file: `../../semantic-conventions.md`                                                          missing-file             remark-validate-links
      58:41-58:83  warning  Link to unknown file: `../../semantic-conventions.md`                                                          missing-file             remark-validate-links
      72:41-72:83  warning  Link to unknown file: `../../semantic-conventions.md`                                                          missing-file             remark-validate-links
      81:41-81:83  warning  Link to unknown file: `../../semantic-conventions.md`                                                          missing-file             remark-validate-links
      90:41-90:83  warning  Link to unknown file: `../../semantic-conventions.md`                                                          missing-file             remark-validate-links

⚠ 34 warnings
```

</details>

After the patch:

<details>

```
$ remark -u validate-links .
CHANGELOG.md: no issues found
CONTRIBUTING.md: no issues found
README.md: no issues found
issue-management.md: no issues found
milestones.md: no issues found
specification/api-distributedcontext.md: no issues found
specification/api-metrics.md: no issues found
specification/api-propagators.md: no issues found
specification/api-resources.md: no issues found
specification/api-sampling.md: no issues found
specification/api-tracing.md: no issues found
specification/concurrency.md: no issues found
specification/data-semantic-conventions.md: no issues found
specification/library-guidelines.md: no issues found
specification/library-layout.md: no issues found
specification/overview.md: no issues found
specification/performance.md: no issues found
specification/sdk-exporter.md: no issues found
work_in_progress/gRPC/gRPC.md: no issues found
work_in_progress/opencensus/HTTP-stats.md: no issues found
work_in_progress/opencensus/HTTP.md: no issues found
work_in_progress/opencensus/StandardResources.md: no issues found
work_in_progress/opencensus/gRPC-stats.md: no issues found
work_in_progress/opentracing/semantic_conventions.md: no issues found
work_in_progress/specification/README.md: no issues found
work_in_progress/specification/encodings/BinaryEncoding.md: no issues found
work_in_progress/specification/encodings/CensusServerStatsEncoding.md: no issues found
work_in_progress/specification/encodings/README.md: no issues found
work_in_progress/specification/resource/Resource.md: no issues found
work_in_progress/specification/trace/LogCorrelation.md: no issues found
work_in_progress/specification/trace/README.md: no issues found
work_in_progress/specification/trace/Sampling.md: no issues found
work_in_progress/specification/trace/Span.md: no issues found
work_in_progress/specification/trace/TraceConfig.md: no issues found
work_in_progress/typedspans/README.md: no issues found
```

</details>

### CODE REVIEW

File: work_in_progress/specification/trace/LogCorrelation.md [hunk 1]
```diff
@@ -23,16 +23,16 @@ span context available in each log entry:
 
 ### Trace ID
 
-The trace ID of the current span.  See [Span.md#traceid](Span.md#traceid).
+The trace ID of the current span.  See [Span.md#traceid](Span.md).
```
Comment (arminru) [inline work_in_progress/specification/trace/LogCorrelation.md]:
```suggestion
The trace ID of the current span. See [SpanContext](../overview.md#spancontext).
```
(working link once it's moved out of the WIP folder)

File: work_in_progress/specification/trace/LogCorrelation.md [hunk 2]
```diff
@@ -23,16 +23,16 @@ span context available in each log entry:
...
 
 ### Span ID
 
-The span ID of the current span.  See [Span.md#spanid](Span.md#spanid).
+The span ID of the current span.  See [Span.md#spanid](Span.md).
```
Comment (arminru) [inline work_in_progress/specification/trace/LogCorrelation.md]:
```suggestion
The span ID of the current span. See [SpanContext](../overview.md#spancontext).
```
(working link once it's moved out of the WIP folder)

File: work_in_progress/specification/trace/LogCorrelation.md [hunk 3]
```diff
@@ -23,16 +23,16 @@ span context available in each log entry:
...
 
 ### Sampling Decision
 
 The sampling bit of the current span, as a boolean.  See
-[Span.md#supported-bits](Span.md#supported-bits).
+[Span.md](Span.md).
```
Comment (arminru) [inline work_in_progress/specification/trace/LogCorrelation.md]:
```suggestion
[SpanContext](../overview.md#spancontext).
```
(working link once it's moved out of the WIP folder)

File: work_in_progress/specification/trace/LogCorrelation.md [hunk 4]
```diff
@@ -23,16 +23,16 @@ span context available in each log entry:
 
 ### Trace ID
 
-The trace ID of the current span.  See [Span.md#traceid](Span.md#traceid).
+The trace ID of the current span. See [SpanContext](../overview.md#spancontext).
```
Comment (reyang) [inline work_in_progress/specification/trace/LogCorrelation.md]: Not sure about this, I thought this document is trying to use two spaces everywhere. https://en.wikipedia.org/wiki/Sentence_spacing

Comment (bogdandrutu) [reply work_in_progress/specification/trace/LogCorrelation.md]: This is not consistent across the entire repo.

**Pull request merged**